lib/controller/controller.go

```diff
@@ -297,6 +307,13 @@ func (ctrl *ProvisionController) Run(stopCh <-chan struct{}) {
 	<-stopCh
 }
 
+// Set the value of failedDeletionRetryThreshold
```

Comment:
linter says comment must begin with `SetFailedDeletionRetryThreshold`

BTW, do we want a `SetFailedProvisionRetryThreshold` too?